anta/tests/routing/bgp.py

```diff
@@ -1226,3 +1226,88 @@ def test(self) -> None:
             self.result.is_success()
         else:
             self.result.is_failure(f"Following BGP peers are not configured or hold and keep-alive timers are not correct:\n{failures}")
+
+
+class VerifyBGPPeerUpdateErrors(AntaTest):
+    """Verifies if the IPv4 BGP peer's update errors counters should be zero.
+
+    Expected Results
+    ----------------
+    * Success: The test will pass if the BGP peer's update error counters are zero.
+    * Failure: The test will fail if the BGP peer's update error counters are non-zero.
```

Comment:
Please update this following the suggestion I made in #778 for the docstring.

Reply:
Updated the docstring.